src/main/scala/mesosphere/marathon/core/appinfo/impl/AppInfoBaseData.scala

```diff
@@ -1,35 +1,42 @@
 package mesosphere.marathon.core.appinfo.impl
 
-import mesosphere.marathon.MarathonSchedulerService
+import java.time.OffsetDateTime
```

Comment:
yes, I'm trying to make intellij generate errors for this so I don't commit them